docs/policy/comanage-instructions-admin.md

```diff
@@ -0,0 +1,39 @@
+Approving COManage Registrations
+================================
+
+OSG is using a new identity management system called COManage.
+This system will be used for managing access to OSG staff-internal monitoring webpages at UNL
+and managing access for OASIS logins.
+
+User registrations must be manually approved by a COManage admin.
+Follow the instructions below to approve a user registration.
+
+!!! note
+    This page is for COManage Admins who want to approve user registrations.
+    If you are a user who wants to register with COManage,
+    go to the [Registering for the OSG COManage](/policy/comanage-instructions-user) page instead.
+
+
+1.  Check for an email from <registry@cilogon.org> saying "Petition for <NAME> changed status from
+    Confirmed to Pending Approval".
+
+1.  Click the first link in the email.
+
+1.  Log in with your institutional credentials.
+    You should see a "View CO Petition..." page.
+    The Status should be "Pending Approval".
+
+1.  Verify that the application is legitimate.
```

Comment:
Do you want to provide any guidance here?

Reply:
I'd love to but I don't have any guidance myself. Anybody else have ideas?

Reply:
Talk to the service owners or the Security team.

Reply:
OK, I added some brief instructions.

Reply:
LGTM